frontend/src/app/libs/event-description/store/event/event.effects.spec.ts

```diff
@@ -2,19 +2,20 @@ import { TestBed, async } from '@angular/core/testing';
 import { Observable, of } from 'rxjs';
 import { EventEffects } from './event.effects';
 import { EventRepository } from '../../services/event-repository.service';
-import { Event } from '../../../landing/models/event';
+import { EventCard } from '../../../landing/models/event';
 import { LoadEvent, LoadEventSuccess, LoadEventFail } from './event.actions';
 import { cold, hot } from 'jasmine-marbles';
 import { Actions } from '@ngrx/effects';
 import { Action } from '@ngrx/store';
 import { createSpy } from '../../../auth/helpers/createSpy';
 import { EventConverter } from '../../services/event-converter.service';
 import { EventDto } from 'src/app/libs/landing/models/event-dto';
+import { EventCardSize } from 'src/app/libs/common-components/components/event-card/models/event-card-size';
 
 describe('Events Effects', () => {
 	let mockEventRepository: jasmine.SpyObj<EventRepository>;
 	let eventConverter: EventConverter;
-	let event: Event;
+	let event: EventCard;
```

Comment:
we loading event from backend, card is only way to represent it, so lets rename `EventCard` to `Event`